cider/tests/utils/QueryArrowDataGenerator.h

```diff
@@ -217,13 +217,14 @@ class QueryArrowDataGenerator {
                               const int32_t null_chance,
                               const int64_t min_len = 0,
                               const int64_t max_len = 0) {
-    CHECK_GE(min_len, 0);
-    CHECK_GE(max_len, min_len);
     std::string col_data = "";
     std::vector<bool> null_data(row_num);
     std::vector<int32_t> offset_data;
     std::mt19937 rng(std::random_device{}());  // NOLINT
-    size_t default_strlen = (max_len < min_len || max_len <= 0) ? 10 : max_len;
+    size_t default_strlen =
+        (max_len < min_len || max_len <= 0 || (max_len - min_len) > 10)
+            ? 10
```

Comment:
I think it's not correct here. It will force the generated varchar length `<=10`. Is that expected?

Reply:
The problem in here is that the ```max_len``` is the same value of ```value_max``` which controls max value of int float..... For other limits of the ```varchar``` length, maybe need add one more argument for ```DataGenerator```

Reply:
I think `varchar(10)` in DDL will not have any limitation on string length in cider internal.

Reply:
Strange, I thought it may impact the results of duckdb since it directly uses DDL to create table. But it seems have no limitation as well.